ui/src/logs/components/OptionsOverlay.tsx

```diff
@@ -134,12 +191,19 @@ class OptionsOverlay extends Component<Props, State> {
       onDismissOverlay,
       onUpdateSeverityFormat,
       onUpdateColumns,
+      onUpdateTruncation,
     } = this.props
-    const {workingLevelColumns, workingFormat, workingColumns} = this.state
+    const {
+      workingLevelColumns,
+      workingFormat,
+      workingColumns,
+      isWorkingTruncated,
+    } = this.state
 
     await onUpdateSeverityFormat(workingFormat)
     await onUpdateSeverityLevels(workingLevelColumns)
     await onUpdateColumns(workingColumns)
+    await onUpdateTruncation(isWorkingTruncated)
```

Comment:
bummer that these are executed in series, but totally not something to fix in this PR :)